.github/workflows/beam_LoadTests_Python_Combine_Dataflow_Batch.yml

```diff
@@ -0,0 +1,116 @@
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Load Tests Combine Dataflow Batch Python
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '40 5 * * *'
+  workflow_dispatch:
+
+#Setting explicit permissions for the action to avoid the default permissions which are `write-all` in case of pull_request_target event
+permissions:
+  actions: write
+  pull-requests: read
+  checks: read
+  contents: read
+  deployments: read
+  id-token: none
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.sha || github.head_ref || github.ref }}-${{ github.event.schedule || github.event.comment.body || github.event.sender.login }}'
+  cancel-in-progress: true
+
+env:
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+  GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
+  GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+
+jobs:
+  beam_LoadTests_Python_Combine_Dataflow_Batch:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      github.event_name == 'schedule' ||
+      github.event.comment.body == 'Run Load Tests Python Combine Dataflow Batch'
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 720
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_LoadTests_Python_Combine_Dataflow_Batch"]
+        job_phrase: ["Run Load Tests Python Combine Dataflow Batch"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - name: Prepare configs
+        id: set_configs
+        shell: bash
+        run: |
+          CURDATE=$(date '+%m%d%H%M%S' --utc)
+          CURCONFIG=$(grep -v "^#.*" ./.github/workflows/load-tests-job-configs/config_Combine_Python_Batch_2GB_10b.txt | tr '\n' ' ')
+          CURCONFIG=$(echo "${CURCONFIG/load-tests-python-dataflow-batch-combine-1-/load-tests-python-dataflow-batch-combine-1-$CURDATE}")
```

Comment:
What is this 2nd echo doing (here and in other configs)?

Reply:
It replaces `job_name` substring with the "name+current date".
In .groovy it was: 
`def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))`
...
`job_name             : "load-tests-python-dataflow-batch-combine-1-${now}"`

Reply:
Could you add an explanatory comment to this section explaining what we are doing? It is hard to gather intent from the code

Reply:
Done